langchain/client/models.py

```diff
@@ -31,12 +31,20 @@ class Example(ExampleBase):
     runs: List[Run] = Field(default_factory=list)
 
 
+class ExampleUpdate(BaseModel):
```

Comment:
1) Should we be setting frozen in the config?

Reply:
Ya